### PR TITLE
Feature/remove variable units sch calculations

### PR DESCRIPTION
**Pull request by jaroldwong**
Issue: https://trello.com/c/u8CDFjVw/2105-variable-units-should-not-be-included-in-sch-calculations-site-wide

### CODE REVIEW

File: app/workloadSummaryReport/directives/workloadTotals/workloadTotals.css [hunk 1]
```diff
@@ -31,3 +31,10 @@
 	padding-left: 20px;
 	font-size: 12px;
 }
+
+.workload-totals__warning {
+	font-size: 10px;
+	padding-bottom: 20px;
+	padding-left: 20px;
+	color: gray;
+}
```
Comment (ltwheeler) [inline app/workloadSummaryReport/directives/workloadTotals/workloadTotals.css]: Missing newline at end of file

File: app/budget/directives/budgetSummary/budgetSummary.css [hunk 2]
```diff
@@ -18,57 +18,64 @@
...
-	text-align: center;
+  text-align: center;
 }
+
+.budget-summary__warning {
+  font-size: 10px;
+  padding-bottom: 20px;
+  padding-left: 20px;
+  color: gray;
+}
```
Comment (ltwheeler) [inline app/budget/directives/budgetSummary/budgetSummary.css]: Missing newline at end of file

File: app/shared/services/SchService.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ * @ngdoc service
+ * @name ipaClientAngularApp.SchService
+ * @description
+ * # SchService
+ * Service in the ipaClientAngularApp.
+ */
+class SchService {
+  constructor() {
+    return {
+      getUnits: function (course) {
```
Comment (ltwheeler) [inline app/shared/services/SchService.js]: The code looks good,
However if I was trying to find a getUnits helper function, I don't think I'd ever think to look for it in the SchService.

My thinking goes something like:
The concepts in our code (courses, units, SCH, sections, terms, schedules, etc) are all related to each other, and it's up to us to determine if we want to architect something as merely a property of a larger concept, or as its own concept.

My first thought is renaming schService to courseService, as they both are closely connected to courses, and for the moment have very little logic surrounding them so they might not need to be separated.

What do you think?

Comment (jaroldwong) [reply app/shared/services/SchService.js]: That sounds sensible- I went ahead and renamed it to CourseService.

**Pull request merged**